reference/library/src/morpheus-master/sass/modules/tool/forums/_forums.scss

```diff
@@ -49,6 +49,18 @@
 			}
 		}
 	}
+	
+	.permissionPanel{
+		input[type="checkbox"]{
+			@media #{$nonphone}{
+				float:left;
+				& + label {
+					overflow:auto;
+					display:block;
+					padding-top: 2px;
+				}
+			}
```

Comment:
Sorry!! I wrote a typo too, is "nonPhone" no "nonphone". It seems that i committed one previous version of the file. I´ll fix it in two minutes